client/src/feedbacker-components/survey/survey.scss

```diff
@@ -1,8 +1,8 @@
 @import '../../scss/compulsory';
-// Atoms
-@import '../../scss/atoms-organisms/button';
-@import '../../scss/atoms-organisms/forms';
-@import '../../scss/atoms-organisms/progress-bar';
+// Atoms & organisms
```

Comment:
split to atoms then organisms with comment

Reply:
Moved atoms before organisms in all files also